sections/sprint-1/experiment.tex

```diff
@@ -126,19 +166,21 @@ \subsection{Possible influences on the test}
 While the experiment was running the tags could throw errors instead of giving a position if something went wrong.
 Mainly during the multi-tag tests, some tags would report back with errors such as unable to get firmware version, flash memory corrupted or an error message saying there was no error.
 At that point in the experiment, it was not clear whether it was a hardware error with some of the tags or if it was code related.
-The tags would throw these errors randomly and then report back their position as normal at the next update.
+The tags would throw these errors randomly and then measure their position normally at the next update.
 This resulted in some of the tags having periods of update rates of less than 0.2 Hz.
 It was decided that further investigation into this issue should be conducted at a later point.
 
 \subsection{Conclusion on the experiment}
-Our conclusion of this experiment is that the precision is satisfactory, but the update rate is not optimal as that does not meet the standards of the refresh rate needed.
+Our conclusion of this experiment is that the precision is satisfactory, but the update rate is not optimal as that does not meet the standards of the refresh rate needed with the current settings.
 \\\\
 We also need to consider the spikes in the coordinates, such that the user does not jump around on the screen, even though the user might not be moving.
+A good solution would be to use a algorithm to correct the data.
 \\\\
 The results of this experiment shows that a three dimensional game is not viable with the errors that we had on the $z$ coordinate. 
-This is due to the height difference between the anchors only being 20 cm.
+This is most likely due to the height difference between the anchors only being 20 cm.
 If we, later on, choose to make use of the $z$ coordinate, then it is necessary to conduct another experiment with the Pozyx with different, larger height differences, but as it currently is only a two-dimensional game it is not necessary.
 \\\\
-It also seemed like some tags were not able to send as many data points as others.
-This was also due to getting errors from some of the tags.
-This is something that should be investigated.
+It also seemed like some tags were not able to send as many measurements as others.
+Consistently through the experiments tag 26895 was the best tag regarding update rate sometimes having a update rate almost 3 times faster than the other tags.
+In part do to some tags seemingly having a slower update rate but also because some of them would report errors instead of measurements randomly.
```

Comment:
```suggestion
This was caused in part due to some tags seemingly having a slower update rate, but also because some of them would report errors instead of measurements randomly.
```